Add OutlineViewer Details Page (#971)
wpilibsuite/frc-docs: source/docs/software/wpilib-tools/outlineviewer/index.rst

OutlineViewer
=============

.. image:: images/outline-viewer.png

OutlineViewer is a utility used to view, modify and add to the contents of the NetworkTables for debugging purposes. It displays all key value pairs currently in the NetworkTables and can be used to modify the value of existing keys or add new keys to the table. OutlineViewer is included in the C++ and Java language updates. Teams may need to install the Java Runtime Environment to use the OutlineViewer on computers not set up for Java programming.

> In Visual Studio Code, press Ctrl + Shift + P and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette and select Start Tool, then select OutlineViewer.

Reviewer: This sentence is a run-on. I would break it out a bit.

Author: which one?

Reviewer: "In Visual Studio Code, press Ctrl + Shift + P and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette and select Start Tool, then select OutlineViewer."

Author: This exact same sentence is used in 5 different places if frc-docs.  I am not saying it is right but I am not sure what it should be.  I will change it for all 5.  Ideas?

Author: In Visual Studio Code, press Ctrl + Shift + P and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette.  Then select Start Tool, and then from the next menu select OutlineViewer.

Author: fixed per discord convo

To connect to your robot, open OutlineViewer and set the “Server Location” to be your team number. After you click start, OutlineViewer will connect.  If you have trouble connecting to OutlineViewer please see the :ref:`Dashboard Troubleshooting Steps <docs/software/driverstation/troubleshooting-dashboard-connectivity:Troubleshooting Dashboard Connectivity>`.

.. image:: images/outlineviewer.png

> To additional key/value pairs to NetworkTables right click on where you want it to go and choose the corresponding data type.

Reviewer: This english seems bad. I think you meant
```suggestion
To add additional key/value pairs to NetworkTables, right click on a location and choose the corresponding data type.
```

> LabVIEW teams can use the Variables tab of the LabVIEW Dashboard to accomplish the same functionality as OutlineViewer.

Reviewer: Maybe this should be a ``.. note::`` admonition?

Author: done
